graphql/queries/challengeLeadOrganisations.graphql

```diff
@@ -0,0 +1,11 @@
+query challengeLeadOrganisations($ecoverseId: UUID_NAMEID!, $challengeID: UUID_NAMEID!) {
+  ecoverse(ID: $ecoverseId) {
+    id
+    challenge(ID: $challengeID) {
+      id
+      leadOrganisations {
+        id
+      }
+    }
+  }
+}
```

Comment:
Both queries can be combined in single one to serve exactly this page. Also the whole object for the leadOrganisations can be loaded, which can be used directly as Leading organizations and just filter out the reset. Also the organizations fields can be defined as a fragment which will resolve the type question.